static/app/components/events/autofix/useAutofixCodebaseIndexing.tsx

```diff
@@ -62,24 +64,32 @@ export function useAutofixCodebaseIndexing({
   const status = apiData?.status ?? null;
   const prevStatus = usePrevious(status);
 
-  const startIndexing = useCallback(() => {
+  const startIndexing = useCallback(async () => {
     if (!projectSlug) {
       return;
     }
 
-    // Triggering the create endpoint on the seer side should make it start return indexing status...
-    api.requestPromise(makeCodebaseIndexCreateUrl(organization.slug, projectSlug), {
-      method: 'POST',
-    });
-
-    // We set it anyways to trigger the polling
     setApiQueryData<CodebaseIndexingStatusResponse>(
       queryClient,
       makeCodebaseIndexStatusQueryKey(organization.slug, projectSlug),
       {
         status: AutofixCodebaseIndexingStatus.INDEXING,
       }
     );
+
+    // Triggering the create endpoint on the seer side should make it start return indexing status...
+    try {
+      await api.requestPromise(
+        makeCodebaseIndexCreateUrl(organization.slug, projectSlug),
+        {
+          method: 'POST',
+        }
+      );
+    } catch {
+      addErrorMessage(
+        t('Something went wrong while Autofix was indexing your codebase.')
```

Comment:
Can we allow a more specific error message to be surfaced from seer? So there can be two ways codebase indexing errors – a 500 from this endpoint would be from initializing the task, stuff like incorrect git setup etc like we just saw, or errors during the indexing itself which can be surfaced from the status endpoint that's polled

Reply:
Added more detailed messaged for the create and status endpoints, this is what the status looks like when errored:
 
![CleanShot 2024-05-21 at 10 39 46](https://github.com/getsentry/sentry/assets/10888943/68a159c2-55e2-42a9-bbe2-90bb4c75a279)